plug.vim

```diff
@@ -1169,7 +1169,7 @@ function! s:job_abort()
       silent! call job_stop(j.jobid)
     endif
     if j.new
-      call s:system('rm -rf ' . plug#shellescape(g:plugs[name].dir))
+      call s:rm_rf(g:plugs[name].dir)
```

Comment:
`s:rm_rf(dir)` cannot run `delete(a:dir, 'rf')` without stalling the build. I guess it fails in the terminal.